src/App.js

```diff
@@ -0,0 +1,6 @@
+import Home from "./Home/index.js";
```

Comment:
While importing from components, you don't have to put `index.js` in the import line.